main.go

```diff
@@ -1982,7 +1983,21 @@ func startTunnel(ctx context.Context, recordsPath string, tunnelInfoPort int) {
 		}
 	}()
 
-	<-ctx.Done()
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
+	select {
+	case <-c:
+	case <-ctx.Done():
+	}
```

Comment:
Can you explain/remind why do we need to create that additional channel c for os signals? I guess ctx.Done() doesn't cover the manual shutdown of the progam, does it?

Reply:
It's meant to hijack Ctrl+C system event and allow gently closing process.